buildspec/kotlinTests.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+    install:
+        runtime-versions:
+            java: corretto8
+
+    build:
+        commands:
+            - |
+                # TODO - iterate through the subdirectories to have one build script per runtime
+                cd telemetry/jetbrains
+                ./gradlew test
+
+artifacts:
+    files:
+        - '**/*'
```

Comment:
wouldn't this be something like `build/dist/...` ? Seems wasteful to dump the entire tree into an artifacts store (and then will also cause big/noisy downloads for anyone who actually is interested in the artifacts)

Reply:
This is a case of bad copy paste, for now we don't care about these artifacts, so removed